collector/cloudwatch/settings.php

```diff
@@ -0,0 +1,88 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Settings for cltr_cloudwatch.
+ *
+ * @package   cltr_cloudwatch
+ * @author    Jason den Dulk <jasondendulk@catalyst-au.net>
+ * @copyright 2022, Catalyst IT
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+if ($hassiteconfig) {
+
+    if ($ADMIN->fulltree) {
+        $settings->add(new admin_setting_heading('cltr_cloudwatch_settings', '',
+            get_string('pluginnamedesc', 'cltr_cloudwatch')));
+
+        // AWS settings.
+        $settings->add(new admin_setting_heading('cltr_cloudwatch_aws',
+            get_string('awssettings', 'cltr_cloudwatch'),
+            get_string('awssettings_desc', 'cltr_cloudwatch')
+        ));
+
+        $regionoptions = [
```

Comment:
in local_aws there is a region selection admin setting admin_settings_aws_region

Reply:
Done